tests/testsuite/namespace.go

```diff
@@ -191,6 +191,9 @@ func CleanNamespaces() {
 		pvs, err := virtCli.CoreV1().PersistentVolumes().List(context.Background(), listOptions)
 		util.PanicOnError(err)
 		for _, pv := range pvs.Items {
+			if pv.Spec.ClaimRef == nil || pv.Spec.ClaimRef.Namespace != namespace {
```

Comment:
Don't we want to clean up the PVs in any case?

Reply:
Since PVs are a cluster scoped resource I think it only makes sense to delete PVs that belong to the namespace being cleaned up.  Why delete statically allocated available PVs?  Current code will delete PVs created by local storage provisioner for example

Reply:
This list gets filtered already https://github.com/kubevirt/kubevirt/blob/6ad91329f2e72aaa6b6c961fdf38496052602621/tests/testsuite/namespace.go#L83-L86